### PR TITLE
Py38

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

**Comment by auvipy** [comment]
will rebase

File: tox.ini [hunk 1]
```diff
@@ -11,7 +11,7 @@ commands=
 # tox -e docs to mimick readthedocs build.
 # as of today, RTD is using python3.6 and doesn't run "setup.py install"
 [testenv:docs]
-basepython=python3.6
+basepython=python3.8
```
Comment (JonathanHuot) [inline tox.ini:14]: have you verified RTD is using python3.8 and no longer python3.6 ? when the comment above was written, it was 3.6, at least :)

Comment (auvipy) [reply tox.ini:14]: i think need to revert this

Comment (JonathanHuot) [reply tox.ini:14]: Can you rever tit ? Then we merge the PR

Comment (auvipy) [reply tox.ini]: this can be squash merge now

**Pull request closed without merging**